app/build.gradle.kts

```diff
@@ -158,4 +158,7 @@ dependencies {
 
     // Markdown
     implementation(libs.markdown.renderer)
+
+    // Icons
+    implementation("br.com.devsrsouza.compose.icons:font-awesome:1.1.0")
```

Comment:
What is the license of the icons *and* the library? non-free icons should be avoided

Reply:
Both the [library](https://github.com/DevSrSouza/compose-icons/blob/master/LICENSE) and [font awesome](https://github.com/DevSrSouza/compose-icons/blob/master/font-awesome/src/commonMain/resources/font-awesome-license.txt) are free to use.